tailcfg/tailcfg.go

```diff
@@ -1747,6 +1748,8 @@ const (
 	CapabilityWakeOnLAN = "https://tailscale.com/cap/wake-on-lan"
 	// CapabilityIngress grants the ability for a peer to send ingress traffic.
 	CapabilityIngress = "https://tailscale.com/cap/ingress"
+	// CapabilitySSHSessionHaul grants the ability to send SSH session logs.
+	CapabilitySSHSessionHaul = "https://tailscale.com/cap/ssh-session-haul"
```

Comment:
why is this a cap at all, as opposed to just its presence in the SSHConfig? If control can send one it can send both, but why two?

Reply:
oh, this is in the inter-node section! carry on. Maybe reword the comment to make that more clear. I see some of them are more clear about that but then the last few got shorter.

Reply:
This is checked on the aggregator to know if the SSH server can send logs to it.